k8s/models/common.py

```diff
@@ -50,6 +50,20 @@ class ObjectMeta(Model):
     selfLink = ReadOnlyField(six.text_type)
     uid = ReadOnlyField(six.text_type)
 
+    
+class ObjectReference(Model):
+    apiVersion = Field(six.text_type)
+    fieldPath = Field(six.text_type)
+    kind = Field(six.text_type)
+    name = Field(six.text_type)
+    namespace = Field(six.text_type)
+    resourceVersion = Field(six.text_type)
+    uid = Field(six.text_type)
+
+    
+class Time(Model):
```

Comment:
This looks strange - what is it for?

Reply:
There is a reference to this object in

CronJobStatus v1beta1 batch

lastScheduleTime Time | Information when was the last time the job was successfully scheduled.

https://github.com/kubernetes/apimachinery/blob/master/pkg/apis/meta/v1/time.go

I am not sure, if that could be imported from somewhere else.

Reply:
I was sure I had commented on this. This is probably poorly documented, but the `Time` type in the Kubernetes API should be represented by python `datetime.datetime`, as can done in `ObjectMeta` in this very file.

Reply:
I am not sure I understand this relation.

class CronJobStatus(Model):
    active = ListField(ObjectReference)
    lastScheduleTime = Field(Time)

Is CronJobStatus behaving same as CronJobList?

Where the Time should be referenced in this case?

Reply:
Basically, every time the Kubernetes API uses the type `Time`, you should use `datetime.datetime` instead. So `CronJobStatus` should be defined like this:

```
class CronJobStatus(Model):
    active = ListField(ObjectReference)
    lastScheduleTime = Field(datetime.datetime)
```

There is no special handling of `Status` objects in the library (we probably should consider that at some point).

Reply:
I have update it.
I also changed the Field to ReadOnlyField, which is applied for datetime.datetime elsewhere.